video/out/vo_gpu_next.c

```diff
@@ -56,6 +56,12 @@
 #include "osdep/windows_utils.h"
 #endif
 
+enum quality_preset {
+    PRESET_FAST,
+    PRESET_DEFAULT,
```

Comment:
very minor but for consistency might be useful to have `PRESET_DEFAULT == 0`

Reply:
sure, I sorted it by the speed/features enabled, but 0 as default makes more sense.